chapter10/kds/ex2.pl

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/perl
+
+use warnings;
+use strict;
+use Scalar::Util qw(looks_like_number);
+
+my $random = int(1 + rand 100);
+my $verbose = 1;
+
```

Comment:
This is confusing. If verbose is true then it can't use the shortcut in the or statement and so prints the debug statement